src/nominatim_db/tools/tiger_data.py

```diff
@@ -90,16 +89,18 @@ async def add_tiger_data(data_dir: str, config: Configuration, threads: int,
     """
     dsn = config.get_libpq_dsn()
 
-    with connect(dsn) as conn:
-        if freeze.is_frozen(conn):
-            raise UsageError("Tiger cannot be imported when database frozen (Github issue #3048)")
-
     with TigerInput(data_dir) as tar:
         if not tar:
             return 1
 
         with connect(dsn) as conn:
             sql = SQLPreprocessor(conn, config)
+
+            if sql.env.globals['db']['reverse_only']:
```

Comment:
To be exact, the TIGER data can be imported in reverse_only as long as the database is not frozen. Only when frozen _and_ in reverse_only mode do we get in trouble.

Accessing the internals of the SQLPreprocessor is clever but might get us in trouble in the future. Better use a separate `table_exists('search_name')` here. It's a bit more expensive but this is a long running function anyway